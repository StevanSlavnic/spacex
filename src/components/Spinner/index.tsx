export default function Spinner() {
  return (
    <div data-testid="spinner" className="flex justify-center items-center">
      <div className="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-gray-800"></div>
    </div>
  );
}
