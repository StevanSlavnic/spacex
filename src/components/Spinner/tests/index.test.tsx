import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import Spinner from "../index";

describe("Spinner Component", () => {
  it("should render without crashing", () => {
    const { container } = render(<Spinner />);
    expect(container).toMatchSnapshot();
  });
});
