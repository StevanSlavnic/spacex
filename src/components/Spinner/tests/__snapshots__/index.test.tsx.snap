// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should render without crashing 1`] = `
<div>
  <div
    class="flex justify-center items-center"
    data-testid="spinner"
  >
    <div
      class="animate-spin rounded-full h-12 w-12 border-t-2 border-b-2 border-gray-800"
    />
  </div>
</div>
`;
